ci-operator/config/openshift/template-service-broker-operator/openshift-template-service-broker-operator-master.yaml

```diff
@@ -0,0 +1,12 @@
+build_root:
+  project_image_build:
```

Comment:
same here.

Reply:
The docs here have project_image_build at https://github.com/openshift/ci-operator/blob/master/ONBOARD.md#source-code-image-target